blocks/large-promo-block/features/large-promo/default.jsx

```diff
@@ -99,12 +139,142 @@ const LargePromoItem = ({ customFields, arcSite }) => {
     }`,
 		}) || null;
 
-	return <LargePromoPresentation content={content} {...customFields} />;
+	const { editableContent, searchableField } = useEditableContent();
+	const { registerSuccessEvent } = useComponentContext();
+	const {
+		dateLocalization: { language, timeZone, dateTimeFormat } = {
+			language: "en",
+			timeZone: "GMT",
+			dateTimeFormat: "LLLL d, yyyy 'at' K:m bbbb z",
+		},
+	} = getProperties(arcSite);
+	const phrases = getTranslatedPhrases(getProperties(arcSite).locale || "en");
+	const bylineNodes = formatAuthors(content?.credits?.by, phrases.t("global.and-text"));
+
+	// show the override url over the content image if it's present
+	// get the image from content if no override
+	const targetImage = imageOverrideURL || getImageFromANS(content);
+
+	// Start Overline data
+	const {
+		display: labelDisplay,
+		url: labelUrl,
+		text: labelText,
+	} = (content?.label && content?.label?.basic) || {};
+	const shouldUseLabel = !!labelDisplay;
+
+	const { _id: sectionUrl, name: sectionText } =
+		(content?.websites &&
+			content?.websites[arcSite] &&
+			content?.websites[arcSite].website_section) ||
+		{};
```

Comment:
Just FYI, this could be:
```suggestion
		content?.websites?.[arcSite]?.website_section || {};
```
Not going to require it though